src/storage/mongodb/BillingStorage.ts

```diff
@@ -26,6 +26,12 @@ export default class BillingStorage {
     return invoicesMDB.count === 1 ? invoicesMDB.result[0] : null;
   }
 
+  public static async getInvoicesInTestMode(tenantID: string): Promise<DataResult<BillingInvoice>> {
+    // Returns all invoices that are to be finalized and paid
+    const invoicesMDB = await BillingStorage.getInvoices(tenantID, { liveMode: false }, Constants.DB_PARAMS_MAX_LIMIT);
```

Comment:
No need an intermediate var:
`return BillingStorage.getInvoices(tenantID, { liveMode: false }, Constants.DB_PARAMS_MAX_LIMIT);`

Reply:
You can also call right away the BillingStorage.getInvoices(...), you will not create a new method whenever you have a filter to call in the main one? :-)